sup.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+
+from brutal.core.plugin import cmd
+from datetime import datetime
+import admanbot.plugins.calendar_sk as calendar_sk
```

Comment:
import <name>.plugins.stuff won't work because of different bot names on different machines
I tried simple `import calendar_sk` but flake8 warned me that it is better to import <name>.plugins.stuff
Going to recode it